python/ray/data/impl/null_aggregate.py

```diff
@@ -85,6 +85,54 @@ def _init(k: KeyType) -> AggType:
     return _init
 
 
+def _null_wrap_merge(
```

Comment:
This looks the existing function, can it not show it as a newly added one to make diff cleaner?

Reply:
Unfortunately not, moving this function up makes sense given the change in argument ordering in `AggregateFn`, and git isn't smart enough to present a nicer diff.